package.json

```diff
@@ -5,6 +5,7 @@
     "scripts": {
         "prepare": "husky",
         "dev": "yarn && nuxi dev",
+        "dev:localdb": "NODE_ENV=development yarn && nuxi dev",
```

Comment:
localdb doesn't make sense here as we're not starting a database. 
I would just add the NODE_ENV variable to the `dev` task